client/my-sites/promote-post-i2/components/search-bar/index.tsx

```diff
@@ -92,15 +98,31 @@ export default function SearchBar( props: Props ) {
 		},
 	];
 
+	const wooPostTypeOptions = [
+		{
+			value: 'product',
+			label: translate( 'Products' ),
+		},
+		{
+			value: 'post',
+			label: translate( 'Posts' ),
+		},
+		{
+			value: 'pages',
```

Comment:
I think this should be `page` (singular). It isn't filtering Ok right now.

Reply:
let me correct this :)

Reply:
Solved